addons/hr_holidays/views/hr_leave_allocation_views.xml

```diff
@@ -355,7 +355,7 @@
         <field name="name">All Allocations</field>
         <field name="res_model">hr.leave.allocation</field>
         <field name="view_mode">tree,kanban,form,activity</field>
-        <field name="context">{}</field>
+        <field name="context">{'block_allocation_type_no':1}</field>
```

Comment:
I give a more meaningfull name to the context. We have a lot of context used everywhere and it is important to know what it does right away. 
We could go with 'force_allocation_type' for example.